samples/apps/read_all.jakt

```diff
@@ -0,0 +1,17 @@
+/// Expect:
+/// - stderr: "So there I was, in the rain, all alone...\n"
+
+function parse_to_string(file_path: String) throws -> String {
+    mut file = File::open_for_reading(file_path)
+    mut builder = StringBuilder::create()
+    for b in file.read_all().iterator() {
+        builder.append(b)
+    }
+    return builder.to_string()
+}
+
+function main(args: [String]) throws {
+    // replace "mystery.txt" with args[1] to be able to run "./readall mystery.txt"
+    let file_data = parse_to_string(file_path: "mystery.txt")
```

Comment:
You accidentally named the data file `mystory.txt` instead of `mystery.txt`, so this will always fail. You probably want to run your tests before submitting a PR.

Reply:
fixed